dali-mqtt-daemon.py

```diff
@@ -63,95 +91,83 @@ def gen_ha_config(light):
     return json.dumps(json_config)
 
 log_format = '%(asctime)s %(levelname)s: %(message)s'
-logging.basicConfig(format=log_format, level=logging.INFO)
+logging.basicConfig(format=log_format, level=logging.DEBUG)
```

Comment:
default should be INFO not DEBUG

Reply:
Kamil you copied code from my dir with not commited code to master. You should always use code from fresh repo.

Reply:
@dgomes  I changed it to INFO, btw maybe good idea is adding script argument `--log-level` and set default to info?

> Kamil you copied code from my dir with not commited code to master. You should always use code from fresh repo.
No, I used fresh code from us repository, I just changed it for debugging purposes and forget about it

Reply:
--log-level is a good idea

Reply:
I created ticket for --log-level #19

Reply:
This is fixed in [master branch on justnet](https://github.com/JustNet/dali2mqtt/blob/master/dali-mqtt-daemon.py#L78)